src/services/apis.rs

```diff
@@ -0,0 +1,52 @@
+use serde_json::Value;
+
+use crate::{
+    models::{ListKeysRequest, ListKeysResponse},
+    routes::{self, CompiledRoute},
+    services::HttpService,
+    types::Response,
+    unwind_response,
+};
+
+#[allow(unused_imports)]
+use crate::types::HttpError;
+
+#[derive(Debug, Clone)]
+pub struct ApiService;
+
+impl Default for ApiService {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl ApiService {
+    #[must_use]
+    pub fn new() -> Self {
+        Self
+    }
+
+    pub async fn list_keys(
```

Comment:
Same thing goes here, documentation of arguments and return types but no example would be good. See `KeyService.create_key` for how its done there.